relay-server/src/extractors/forwarded_for.rs

```diff
@@ -3,11 +3,25 @@ use std::net::SocketAddr;
 
 use axum::extract::{ConnectInfo, FromRequestParts};
 use axum::http::request::Parts;
+use axum::http::HeaderMap;
 
 #[derive(Debug)]
 pub struct ForwardedFor(String);
 
 impl ForwardedFor {
+    const FORWARDED_HEADER: &str = "X-Forwarded-For";
+    const VERCEL_FORWARDED_HEADER: &str = "X-Vercel-Forwarded-For";
+
+    // We prefer the vercel header because the normal one could get overwritten as explained here.
+    // `https://vercel.com/docs/concepts/edge-network/headers#x-vercel-forwarded-for`
```

Comment:
```suggestion
    /// We prefer the Vercel header because the normal one could get overwritten as explained here.
    /// `https://vercel.com/docs/concepts/edge-network/headers#x-vercel-forwarded-for`
```

Make this a doc comment please.